torch/ao/quantization/pt2e/qat_utils.py

```diff
@@ -218,7 +219,7 @@ def _quantized_qat_conv_bn_pattern(
         x = x / scale_factor.reshape(bias_shape)
         if has_bias:
             x = x + kwargs["conv_bias"].reshape(bias_shape)
-        x = F.batch_norm(x, bn_running_mean, bn_running_var, bn_weight, bn_bias, training=True, eps=bn_eps)
+        x = F.batch_norm(x, bn_running_mean, bn_running_var, bn_weight, bn_bias, training=bn_is_training, eps=bn_eps)
```

Comment:
can you do ignore literals in matching and replacement?

Reply:
Hi @jerryzh168, thanks for the comment. Did you mean set `ignore_literals` as True in `replace_pattern_with_filters`. If so, I think it's already set as 
https://github.com/pytorch/pytorch/blob/77c4565d58fb9c82bd59ebbd5e22471cf8ecafe2/torch/ao/quantization/pt2e/qat_utils.py#L733-L740

However, here for `F.batch_norm` when `training` parameter is `True` or `False` actually generates 2 different aten operator after export as `torch.ops.aten._native_batch_norm_legit.default` and `torch.ops.aten._native_batch_norm_legit_no_training.default`. Will `ignore_literals` still work in this case?